package_check.sh

```diff
@@ -715,7 +715,7 @@ TEST_RESULTS () {
 		# Get the name of the app from the repository name.
 		app_name="$(basename --multiple --suffix=_ynh "$app_arg")"
 		# Extract the app part from the json file. From the name line to the url line.
-		json_app_part=$(sed -n "/$app_name/,/${app_arg//\//.}/p" "$script_dir/list.json")
+		json_app_part=$(sed -n "/${app_name}/,/${app_arg//\//.}/{p; /${app_arg//\//.}/q}" "$script_dir/list.json")
```

Comment:
Ok, since the idea is to match the first `}`
This works and it's easier.
```
json_app_part=$(sed -n "/$app_name/,/}/p" "$script_dir/list.json")
```
That's just about to match } instead of the url.

But, there's also the problem about gitlab-runner.
So, to fix that we need to match only the complete name of the app. By matching as well the double quotes.
```suggestion
		json_app_part=$(sed -n "/\"$app_name\"/,/}/p" "$script_dir/list.json")
```

Reply:
Easier and working!